test/contributionreward.js

```diff
@@ -515,6 +515,58 @@ contract('ContributionReward', accounts => {
      var reputationGainAsProposer = proposingRepRewardConstA;
      assert.equal(reputation, 1000+reputationGainAsVoter + reputationGainAsProposer);
     });
+
+    it("execute proposeContributionReward via genesisProtocol and redeem using Redeemer for un excuted boosted proposal", async function() {
+      var standardTokenMock = await ERC20Mock.new(accounts[0],1000);
+      var testSetup = await setup(accounts,true,standardTokenMock.address);
+      var reputationReward = 12;
+      var nativeTokenReward = 12;
+      var ethReward = 12;
+      var periodLength = 0;
+      var numberOfPeriods = 1;
+      //send some ether to the org avatar
+      var otherAvatar = await Avatar.new('otheravatar', helpers.NULL_ADDRESS, helpers.NULL_ADDRESS);
+      await web3.eth.sendTransaction({from:accounts[0],to:testSetup.org.avatar.address, value:20});
+      var tx = await testSetup.contributionReward.proposeContributionReward(testSetup.org.avatar.address,
+                                                                     web3.utils.asciiToHex("description"),
+                                                                     reputationReward,
+                                                                     [nativeTokenReward,ethReward,0,periodLength,numberOfPeriods],
+                                                                     testSetup.standardTokenMock.address,
+                                                                     otherAvatar.address
+                                                                   );
+      //Vote with reputation to trigger execution
+      var proposalId = await helpers.getValueFromLogs(tx, '_proposalId',1);
+
+      await testSetup.contributionRewardParams.votingMachine.genesisProtocol.vote(proposalId,1,0,helpers.NULL_ADDRESS,{from:accounts[1]});
+      await standardTokenMock.approve(testSetup.contributionRewardParams.votingMachine.genesisProtocol.address,1000);
+      await testSetup.contributionRewardParams.votingMachine.genesisProtocol.stake(proposalId,1,1000);
+      await helpers.increaseTime(60+1);
+      var arcUtils = await Redeemer.new(testSetup.contributionReward.address,testSetup.contributionRewardParams.votingMachine.genesisProtocol.address);
+      var redeemRewards = await arcUtils.redeem.call(proposalId,testSetup.org.avatar.address,accounts[0]);
```

Comment:
why do you call .call first instead of just executing the transaction?

Reply:
nothing special , just checking the call works for the case there is no Dao bounty .